tests/test_filters.py

```diff
@@ -36,7 +36,12 @@ def test_transactions(db, fname, backend):
     assert_equal(list(cal), exp_head)  # Check we have the correct headers
 
     # SimId et al. change at each test need to drop it
-    drop_cols = ['SimId', 'TransactionId', 'ResourceId', 'ReceiverId', 'SenderId']
+    drop_cols = [
+        'SimId',
```

Comment:
hmmmm... A little strange that it forces these to all be on their own line, but it doesn't do that in line 162 of the previous file above.

Reply:
I changed this a little, this should look better now

Reply:
It was OK - just oddly inconsistent, in my mind.  Is this still PEP8 compliant?

Reply:
yes that's one of the advantage of `autopep8` (and might be an inconvenient sometimes), sometimes there is multiple way to have it pep8 compliant, and then `autopep8` will not change it